cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1751,16 +1751,6 @@
   </property>
 
   <property>
-    <name>metrics.messaging.fetcher.limit</name>
```

Comment:
This is a deprecated config. You should move this block into `cdap-default-deprecated.xml` file